### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by jim-rosales** [r-anga99/prj-rev-bwfs-dasmoto#6]
Grade: Meets Expectations

Summary:
Congrats on finishing the Dasmoto project! You did an excellent job applying your CSS efficiently as I didn't see any duplicate styles in your stylesheet. Apart from this, you did an excellent job structuring your HTML code by making use of the elements you've learned so far and of classes along with ids. 

The only area I felt your project can improve on is regarding formatting. Consider reading over this [article](https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html/) that covers HTML formatting in depth and how to properly indent nested elements. You can also take a look at this [gist](https://gist.github.com/jim-rosales/c1446e66054c97071406610be771fa79) that follows most of the suggestions on the article for formatting so you can compare your code with the solution.

**Comment by r-anga99** [comment]
Cool - thanks heaps for that - in regards to formatting - if you had multiple <div>'s within a <div> would you indent them with each new one you add? so the code would look more diagonal as opposed to vertical?
New to this so might not come across right!
Angus

**Comment by r-anga99** [comment]
haha - the code that didn't show up was meant to be the division tag!

**Comment by jim-rosales** [comment]
Yup! the W3C recommends two spaces for indentation for each nested element, so if you had two nested `div`s they would look like this:

```HTML
<div class="first">
  <div class="second">
    <div class="third">
    </div>
  </div>
</div>
```

Formatting it this manner just makes it easier to determine which closing tag `</div>` goes with which `<div>`

Let me know if this clarifies the question you had!

**Comment by jim-rosales** [comment]
I also updated my original message since it looks like I forgot to include the sample gist with the HTML solution for the project

**Comment by r-anga99** [comment]
ah legend!! Thanks for that mate - answered my question very well!
